### ISSUE TITLE
Robot.pause() preceding module command is skipped

### ISSUE THREAD

**Issue by ahiuchingau** [Opentrons/opentrons#2811]
## overview

Robot.pause() does not behave as expected when paired with module commands.

## current behavior

```
robot.pause('Something something')
mag_module.engage()
p50.delay(minutes=5)
```
Robot will not pause, and go straight to engage the module.

## expected behavior

Robot should wait for user to resume before executing module commands.

**Comment by mcous** [comment]
@ahiuchingau do you know if this behavior is present in PAPIv2?

**Comment by ahiuchingau** [comment]
@mcous It is present in PAPIv2 as well

**Comment by SyntaxColoring** [comment]
See also #4801.

**Comment by sfoster1** [comment]
Should be fixed by #4870

**Comment by sfoster1** [comment]
Closed by #4870
